sample/src/main/java/ro/nttdata/ligaaclabs/sample/api/HelloResource.java

```diff
@@ -0,0 +1,47 @@
+package ro.nttdata.ligaaclabs.sample.api;
+
+import javax.ejb.Stateless;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.QueryParam;
+import javax.ws.rs.core.Context;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.UriInfo;
+
+/**
+ * This class provides the REST endpoint for performing hello calls.
+ *
+ */
+@Path("/status/hello")
+@Stateless
+public class HelloResource {
```

Comment:
Resursa pt hello ar putea fi inlocuita cu o resursa de Ping specifica pt dojo management - de facut in pasul urmator :)